pages/bookmarks/index.js

```diff
@@ -1,12 +1,32 @@
+/* import { set } from "mongoose"; */
 import styled from "styled-components";
+/* import { useState, useEffect } from "react"; */
+
+export default function BookmarksPage(/* {isBookmarked} */) {
+  /*   const [markedPage, setMarkedPage] = useState([]);
+
+  useEffect(() => {
+    if (isBookmarked.length === 0) {
+      setMarkedPage([]);
+    } else {
+      fetch(`${isBookmarked.join()}`)
+        .then((response) => response.json())
+        .then((markedPage) => setMarkedPage(markedPage));
+    }
+  }, [isBookmarked]); */
```

Comment:
Please delete code you don't use anymore.

Reply:
thanks for your review. As mentioned above, I need the greyed out sections in my next step. If you dont mind, I will keep them.

Reply:
You should write your code you need for your next steps in another feature branch.